### PR TITLE
Fixes #31784 - Allow foreman_rails_t to connect to Qpid

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: katello.te [hunk 1]
```diff
@@ -55,6 +55,9 @@ allow tomcat_t katello_candlepin_port_t:tcp_socket name_connect;
 # Candlepin Event Listener connects to Artemis
 allow foreman_rails_t candlepin_activemq_port_t:tcp_socket name_connect;
 
+# Katello connection to qpid for katello agent
+allow foreman_rails_t amqp_port_t:tcp_socket name_connect;
```
Comment (ehelms) [inline katello.te:59]: Does `amqp_port_t` need to be added to the require block?

Comment (jturel) [reply katello.te:60]: Yep, fixed

**Comment by ehelms** [comment]
@ekohl or @evgeni mind checking this as well? I only trust me selinux knowledge so far.

**Comment by evgeni** [comment]
looks reasonable enough to me

**Review by ehelms — APPROVED** [review]

File: katello.te (hunk 1 above)
Comment (lzap) [reply katello.te:60]: You could have written `corenet_tcp_connect_amqp_port(foreman_rails_t)` but this will do.

**Pull request merged**